composer.json

```diff
@@ -7,7 +7,12 @@
             "email": "whitezo@webtown.hu"
         }
     ],
-    "require": {},
+    "require": {
+        "mockery/mockery": "0.9.*",
```

Comment:
A tesztekhez szükséges függőségeket a `require-dev`-be tedd. A `mockery/mockery` és a `phpunit/phpunit` nem szükségesek a használatához.